### ISSUE TITLE
Email: Intopia Feedback - WCAG 3.0 FPWD (5 - Processes)

### ISSUE THREAD

**Issue by jspellman** [w3c/wcag3#280]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0015.html): 

FPWD: Is this approach of using complete processes as the smallest unit of conformance workable for different types of organizations? These include organizations with very large, dynamic, or complex content, medium-sized organizations relying on external accessibility resources, and very small organizations with limited resources?

We believe that the approach can be workable across different types of organisations, as it allows them to focus on making core tasks accessible rather than individual pages, something that should be achievable regardless of organisation size.

Some additional clarity on the definition of a process is required. Processes can be complex (including across multiple pages of a website or views of an application).

Example: For a consumer merchant website that has an ‘Add to Cart’ and ‘Checkout’ step, would this be considered a single process or a number of processes? Depending on interpretation, only part of these steps could be accessible, but the organisation may still be eligible for Bronze status.

**Comment by hidde** [comment]
The issues described in Intopia's email are related to the first public working draft of WCAG 3 and are no longer relevant to the [latest draft that was published 10 September](https://www.w3.org/TR/wcag-3.0/). As issues have either been addressed or are no longer relevant, I am closing this issue.
